static/module/list.js

```diff
@@ -42,21 +60,34 @@ export class List {
   heading = document.querySelector(`[${this.attr}='heading']`)
   list = document.querySelector(`[${this.attr}='list']`)
   hideme = document.querySelectorAll(`[${this.attr}='hideme']`)
+  rangeIndicator = document.querySelector(`[${this.attr}='range']`)
+  pageIndicator = document.querySelector(`[${this.attr}='page']`)
```

Comment:
That is a very strange way to query for what are actually sub elements of our component which is the `section`.  I only ever add to this.

Reply:
Refactoring will come later